graphql/handler/apollofederatedtracingv1/tracing.go

```diff
@@ -8,18 +8,35 @@ import (
 	"google.golang.org/protobuf/proto"
 
 	"github.com/99designs/gqlgen/graphql"
+	"github.com/vektah/gqlparser/v2/gqlerror"
+)
+
+const (
+	ERROR_MASKED     = "masked"
+	ERROR_UNMODIFIED = "all"
+	ERROR_TRANSFORM  = "transform"
 )
 
 type (
 	Tracer struct {
 		ClientName string
 		Version    string
 		Hostname   string
+		Errors     TraceErrors
 	}
 
 	treeBuilderKey string
 )
 
+type TraceErrors struct {
```

Comment:
```suggestion
type TraceErrorOptions *struct {
```
`TraceErrors` implies that it contains one or more errors, but this is actually just the options. The word Trace might actually be redundant, so `ErrorOptions` would also be a good name.